src/peergos/shared/crypto/asymmetric/curve25519/Ed25519SecretKey.java

```diff
@@ -43,10 +44,16 @@ public CborObject toCbor() {
         return new CborObject.CborList(Arrays.asList(new CborObject.CborLong(type().value), new CborObject.CborByteArray(secretKey)));
     }
 
+    @Override
```

Comment:
What about adding a doc-string here 
``@returns message with signature prepended``

Reply:
there is one on the interface